config/xrdsst.yml

```diff
@@ -24,6 +24,8 @@ security_server:
   ssh_private_key: <SSH_PRIVATE_KEY_OS_ENV_VAR_NAME>
   tls_certificates:
     - /path/to/tls_cert
+  certificate_management_hash:
```

Comment:
I suggest renaming this to just "certificate_management" as <CERTIFICATE_MANAGEMENT_HASH> is there already any way and maybe in the future we would like to add something other than hashes there as well